src/views/Dashboard.vue

```diff
@@ -167,6 +171,20 @@ export default {
     /** The NR Number string. */
     nrNumber (): string {
       return sessionStorage.getItem('NR_NUMBER')
+    },
+
+    /** Checks if changes need to be disabled */
+    disableChanges () : boolean {
+      if (this.nrNumber) {
+        return !this.hasCompletedIncorporationFiling
+      } else {
+        return this.hasBlockerFiling
```

Comment:
This could be compacted it a bit, but i'm just nitpicking :) 

`return this.nrNumber ? !this.hasCompletedIncorporationFiling : this.hasBlockerFiling`

Reply:
I like it more compact for this :) good call. 

Updated.